### ISSUE TITLE
Radiator model massDynamics = energyDynamics

### ISSUE THREAD

**Issue by arnoutaertgeerts** [open-ideas/IDEAS#333]
In the [`RadiatorEN442`](https://github.com/open-ideas/IDEAS/blob/develop/IDEAS/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo) model the massDynamics parameter is set equal to the energy dynamics:

```modelica
Fluid.MixingVolumes.MixingVolume[nEle] vol(
    redeclare each package Medium = Medium,
    each nPorts = 2,
    each V=VWat/nEle,
    each final m_flow_nominal = m_flow_nominal,
    each final energyDynamics=energyDynamics,
    each final massDynamics=energyDynamics,
    each final p_start=p_start,
    each final T_start=T_start,
    each final X_start=X_start,
    each final C_start=C_start,
    each final mSenFac=mSenFac) "Volume for fluid stream"
```

Shouldn't you be able to set these separately?  If I understand correctly, this model is copied from Annex60?

**Comment by Mathadon** [comment]
This should indeed be possible. Will you make an issue for this or shall I?

Op 10-apr.-2015 om 16:50 heeft arnoutaertgeerts <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


In the RadiatorEN442<https://github.com/open-ideas/IDEAS/blob/develop/IDEAS/Fluid/HeatExchangers/Radiators/RadiatorEN442_2.mo> model the massDynamics parameter is set equal to the energy dynamics:

Fluid.MixingVolumes.MixingVolume[nEle] vol(
    redeclare each package Medium = Medium,
    each nPorts = 2,
    each V=VWat/nEle,
    each final m_flow_nominal = m_flow_nominal,
    each final energyDynamics=energyDynamics,
    each final massDynamics=energyDynamics,
    each final p_start=p_start,
    each final T_start=T_start,
    each final X_start=X_start,
    each final C_start=C_start,
    each final mSenFac=mSenFac) "Volume for fluid stream"

Shouldn't you be able to set these separately? If I understand correctly, this model is copied from Annex60?

**Comment by arnoutaertgeerts** [comment]
You can do it, where should it actually be submitted?

**Comment by Mathadon** [comment]
On Annex60 github since that's where we merge the model in from:
https://github.com/iea-annex60/modelica-annex60/pull/215

**Comment by Mathadon** [comment]
When the change has been merged, I'll need to make a new merge from Annex 60 into IDEAS.

**Comment by Mathadon** [comment]
This has been merged on the Annex60 library. I'm waiting for some other pull requests before merging Annex 60 again into IDEAS.

**Comment by Mathadon** [comment]
This has been merged in #350
